simple-service-webapp-rest/src/main/java/com/example/rest/AuthenticationEndpoint.java

```diff
@@ -38,8 +38,10 @@ private void authenticate(String username, String password) throws Exception {
         if(username.equals("abcde"))
             if(password.equals("12345"))
                 return;
-        else
-            throw new Exception();
+        else {
+                System.out.println("ham");
+                throw new Exception();
```

Comment:
Should hardcode username an password as static final fields, invalid password should respond with 403